packages/access-api/src/service/index.js

```diff
@@ -18,13 +22,50 @@ export function service(ctx) {
     },
 
     account: {
-      info: Server.provide(Account.info, async ({ capability }) => {
+      info: Server.provide(Account.info, async ({ capability, invocation }) => {
         const results = await ctx.kvs.accounts.get(capability.with)
         if (!results) {
-          throw new Failure('Account not found...')
+          return new Failure('Account not found.')
```

Comment:
It feels counter-intuitive to return an error...why can we not throw these?

Reply:
These are known errors that should be returned to the user but not sent to sentry.
Ucanto supports multiple invocations in the same request thats why the responses are always arrays, so some invocations can succeed and some can error thats why we dont throw here.

In ucanto a known error like "account not found" is a valid response, the server types also include those. Any other exception error like "fail to parse json" that we want to send to sentry we should throw from the code.